index.js

```diff
@@ -70,232 +63,224 @@ module.exports.ProfileHooker = async (settings = { username: "", cookie: "", tim
 
 
 module.exports.PostHooker = async (settings = { url: "", cookie: "", timeout: 50, output: "" }) => {
-    const options = {
-        url: settings.url,
-        cookie: `sessionid=${settings.cookie};`,
-        timeout: settings.timeout ? settings.timeout : 50,
-        output: settings.output || "instahooker-out"
-    }
-
-    const posts = [];
+  const options = {
+        "url": settings.url,
+        "cookie": `sessionid=${settings.cookie};`,
+        "timeout": settings.timeout ? settings.timeout : 50,
+        "output": settings.output || "instahooker-out"

```

Comment:
Remove double quotes from object keys, they're unnecessary.

Reply:
> 

they have been changed to namespace I don't need to remove